org-formation/800-redirects/_tasks.yaml

```diff
@@ -35,3 +37,22 @@ FinopsApiRedirect:  # redirect finops-api.sageit.org to lambda-mips-api cloudfro
     SourceHostedZoneId: "Z0478495257GEB73WFM63"
     # the value of the CNAME record
     TargetHostName: !CopyValue [!Sub '${primaryRegion}-${resourcePrefix}-mips-microservice-CloudfrontDomain', !Ref AdminCentralAccount]
+
+
+# *.app.sagebionetworks.org
+
+DcaDevAppRedirect:  # redirect dca-test.app.sagebionetworks.org to data_curator-infra ALB
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.6.4/templates/R53/cname.yaml
+  StackName: !Sub '${resourcePrefix}-finops-api-redirect'
```

Comment:
finops?

Reply:
whoops, I guess I needed a little more coffee this morning